app/knights_fighting/tournament.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from app.knights_fighting.knight import Knight
+
+
+class Tournament:
+    def __init__(
+            self,
+            knights: dict,
+            opponents_board: list[tuple[str, str]]
+    ) -> None:
+        """
+        opponents_board - List[Tuple[opponent_name, opponent_name]]
+
+        All opponent names must be contained in knights.keys()

```

Comment:
Rewrite docstrings according to [conventions](https://peps.python.org/pep-0257/) with each argument starting from a new line.